psalm.xml.dist

```diff
@@ -5,6 +5,7 @@
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     findUnusedPsalmSuppress="true"
+    errorBaseline="psalm-baseline.xml"
```

Comment:
I provided suggestions on how to get rid of this one - I really don't want to start introducing a baseline for a component that was cleanly designed with zero errors from scratch :D